appeals/e2e/cypress.config.js

```diff
@@ -46,6 +46,7 @@ module.exports = defineConfig({
 		defaultCommandTimeout: 30000,
 		pageLoadTimeout: 60000,
 		experimentalModifyObstructiveThirdPartyCode: true,
+		experimentalRunAllSpecs: true,
```

Comment:
Why was this change included?

Reply:
this is to make it so that we can run all tests in 1 go if needed